src/BlazorUI/Bit.BlazorUI.Extras/wwwroot/leaflet/leaflet-1.9.4.css

```
@@ -0,0 +1,661 @@
+	behavior: url(#default#VML);
+	display: inline-block;
+	position: absolute;
+	}
+
+
+/* control positioning */
+
+.leaflet-control {
+	position: relative;
+	z-index: 800;
+	pointer-events: visiblePainted; /* IE 9-10 doesn't have auto */
```

Stylelint (17.11.1)

[error] Expected "visiblePainted" to be "visiblepainted" (value-keyword-case)

(value-keyword-case)